configdialog.cpp

```diff
@@ -82,7 +84,9 @@ BOOL ConfigDialog::command(WPARAM wparam, LPARAM lparam) {
 BOOL ConfigDialog::slider_changed(WPARAM wparam, HWND slider) {
 	int value = SendMessage(slider, TBM_GETPOS, 0, 0);
 
-	ConfigOption opt = reverse_lookup(slider_ids, GetDlgCtrlID(slider));
+	ConfigOption opt = *std::find_if(Opts::All.begin(), Opts::All.end(), [&](const ConfigOption &opt) {
+		return opt.dialog_control_id == GetDlgCtrlID(slider);
+	});
```

Comment:
One example of things being slightly nicer because of this change. The code looks more complicated, but it allows us to completely get rid of this hacky `reverse_lookup` template.